storage_backends.py

```diff
@@ -43,12 +48,49 @@ def __init__(self, *args, **kwargs):
 
         self.client = dropbox.Dropbox(self.config['access_token'])
 
-    def retrieve(self):
+    def retrieve(self, user):
         _, response = self.client.files_download(self.config['path'])
 
         return todotxtio.from_string(response.content.decode('utf-8'))
 
-    def store(self, todos):
+    def store(self, todos, user):
         data = todotxtio.to_string(todos).encode('utf-8')
 
         self.client.files_upload(data, self.config['path'], mode=dropbox.files.WriteMode('overwrite'), mute=True)
+
+
+def _safe_username(username):
+    return username.replace("/", "_")
```

Comment:
What's the purpose of this method?

Reply:
To prevent users from using the username to access files they should not if for some reason the webdav auth is broken.

Reply:
Well that's a good point.